test/common/network/lc_trie_speed_test.cc

```diff
@@ -15,6 +17,16 @@ std::unique_ptr<Envoy::Network::LcTrie::LcTrie<std::string>> lc_trie_nested_pref
 
 namespace Envoy {
 
+static void BM_LcTrieConstruct(benchmark::State& state) {
```

Comment:
Can you benchmarks for construct with:
- nested prefixes (1420μs before vs 980μs after, i.e. ~30% faster),
- `0.0.0.0/0` (390μs before vs 500ns after, i.e. ~800x faster).